### DISCUSSION TITLE
Typegraphql-prisma roadmap / future releases

### DISCUSSION THREAD

**Discussion by roy-c-git** [MichalLytek/typegraphql-prisma#355] [General]
Hi, 

I'm looking for some sort of roadmap / future release plans for typegraphql-prisma. Specifically, I'm interesting in whether there are plans to update typegraphql-prisma to use the latest version of Apollo (V4)?

Thanks!

**Comment by MichalLytek** [comment]
`typegraphql-prisma` as well as `TypeGraphQL` works with any GraphQL server. You may want to install `type-graphql@2.0.0-beta.1`
